addons/web/static/src/js/views/kanban/kanban_record_quick_create.js

```diff
@@ -294,6 +308,17 @@ var RecordQuickCreate = Widget.extend({
 
         this.cancel();
     },
+    /**
+     * Ignore window click if autocomplete is open and user clicks outside
+     *
+     * @private
+     */
+    _onBodyMousedown(ev) {
+        if ($(document.activeElement).hasClass('ui-autocomplete-input')
+            && $(document.activeElement).autocomplete("widget").is(":visible")) {
```

Comment:
I don't like this at all. It seems very specific to your bug report. What if we want to select something that isn't a many2one? Also, we are in kanban_record, we don't care of what the DOM of our fields is. Couldn't you simply detect we did a mousedown inside the card ?

Reply:
@aab-odoo Well, actual issue is different, dialog.xml is lazy loaded, when write something in many2one first time which doesn't exist and click outside then it will try to load dialog.xml, loading of xml don't waits and dialog is displayed as soon as we click outside, due to which blank dialog is displayed(without content), so to fix this issue we did a workarodund solution, honestly I also don't like this code but as this is workaround so we did like this.

Reply:
@msh-odoo Right, I mixed up this PR and another one, sorry. I'm not sure this is a bug. You click outside the card, it closes itself. It is the last click you do, so it overrides the previous actions you could have done (in this case, opening search more). This is the logic applied everywhere in the webclient: if you click on a menu, says A, and then on another menu, say B before A is loaded, we forget about A and directly open B.